backend/src/events/controllers.ts

```diff
@@ -353,12 +366,12 @@ const deleteAttendee = async (
   userID: string,
   cancelationMessage: string
 ) => {
-  // grabs the user and their email for SendGrid fucntionality
+  // grabs the user and their email for SendGrid functionality
   const user = await userController.getUserByID(userID);
   var userEmail = user?.email as string;
 
   // sets the email message
-  const emailHtml = "USER REMOVED FROM THIS EVENT";
+  const emailHtml = "<b>htmlRegCancel</b>";
```

Comment:
I don't think this will work. This is simply a string not an html file. Did you test this?